pkgs/applications/science/engineering/strictdoc/default.nix

```
@@ -0,0 +1,104 @@
+{ lib
+, stdenv
+, buildPythonApplication
+, fetchFromGitHub
+, python3
+, pythonOlder
+, html5lib
+, invoke
+, openpyxl
+, poetry-core
+, tidylib
+, beautifulsoup4
+, dataclasses
+, datauri
+, docutils
+, jinja2
+, lxml
+, markupsafe
+, pygments
+, reqif
+, setuptools
+, textx
+, xlrd
+, XlsxWriter
+, pytestCheckHook
+}:
+
+buildPythonApplication rec {
+  pname = "strictdoc";
+  version = "0.0.25";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "${pname}-project";
+    repo = pname;
```

```suggestion
    owner = "strictdoc-project";
    repo = "strictdoc";
```
Please don't replace every string that matches pname. It only makes the code harder to read and understand.